sormas-backend/src/main/java/de/symeda/sormas/backend/sormastosormas/share/ShareDataBuilderHelper.java

```diff
@@ -98,7 +98,8 @@ public void pseudonymiePerson(PersonDto personDto, boolean pseudonymizedPersonal
 	public ContactDto getContactDto(Contact contact, Pseudonymizer pseudonymizer) {
 		ContactDto contactDto = contactFacade.convertToDto(contact, pseudonymizer);
 
-		contactDto.setReportingUser(null);
+		// reporting user is not set to null here as it would not pass the validation
```

Comment:
Please create a ticket to cleanup the reporting user validation.

It's strange that only contact validation fails if it has no reporting user.

Reply:
filed #9701